src/main/AuthenticationRequest.ts

```diff
@@ -192,7 +192,11 @@ function createClaimsPayload(opts: SIOP.ClaimOpts): ClaimPayload {
           // There can only be one definition in the vp_token according to the spec
           throw new Error(SIOPErrors.REQUEST_CLAIMS_PRESENTATION_DEFINITION_NOT_VALID);
         } else {
-          vp_token = { presentation_definition: def.definition };
+          vp_token = {
+            nonce: 'NONCE_STRING',
```

Comment:
Hmzz, there needs to be a TODO here, since the nonce is not static of course

Reply:
added